src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -186,6 +193,66 @@ public function getColumnStyle($columnIndex)
         return $this->getStyle();
     }
 
+    /**
+     * Sets the width for a column.
+     *
+     * @param int        $columnIndex Column index.
+     * @param int|string $width       Column with in characters. Set to 'auto' to auto-fit the content.
```

Comment:
Instead of using `auto`, we could just use `0` or `-1`, right? I think I prefer to use `-1` instead.